src/middleware/fileupload.js

```diff
@@ -1,56 +1,60 @@
 import fs from "fs";
-import model from '../db';
+import db from '../db';
 import path from "path";
-
+import _ from 'lodash';
+import config from '../config.json'
 module.exports = {
 
-    fileUpload: function(form, req, res) {
-        return new Promise((reslove, reject) => {
-            form.parse(req, function(err, field, file) {
-                var oldpath = file.file.path;
-                var newpath = path.join(__dirname, './uploads/' + file.file.name);
-                fs.rename(oldpath, newpath, function(err) {
-                    if (err) throw err;
-                    else {
-                        var lineReader = require('readline').createInterface({
-                            input: fs.createReadStream(newpath)
-                        });
-                        var count = 0;
-                        var line_array = [];
-                        var filtered_data = [];
+    fileUpload: function(req, res) {
+        return new Promise((resolve, reject) => {
+            var oldpath = config.path;
+            var newpath = path.join(__dirname, './uploads/AGL_001.TXT');
+            fs.rename(oldpath, newpath, function(err) {
```

Comment:
why is rename required?

Reply:
sir rename picks the old path and stores the file to the new path from where it can be read